app/models/form/lettings/subsections/setup.rb

```diff
@@ -10,14 +10,22 @@ def pages
     @pages ||= [
       organisation_page,
       stock_owner_page,
+      Form::Lettings::Pages::MinRentValueCheck.new("stock_owner_min_rent_value_check", nil, self),
+      Form::Lettings::Pages::MaxRentValueCheck.new("stock_owner_max_rent_value_check", nil, self),
```

Comment:
teh rent soft validations can get triggered by the stock owner?

Reply:
yep, it owning org provider type affects lettype which affects rent ranges and hence the validation triggering/not